src/s_loader.c

```diff
@@ -329,7 +329,7 @@ int sys_load_lib(t_canvas *canvas, const char *classname)
 
     if (sys_onloadlist(classname))
     {
-        error("%s: already loaded", classname);
+        /*error("%s: already loaded", classname);*/
```

Comment:
i'm not a big fan of disabling code by commenting it.
why don't you just delete the line?

Reply:
Yeah. I say remove it altogether.

Reply:
you're right it's not nice.
I was embarrassed because actually Miller changed a "verbose(1, ...)" for an "error" some months ago (fee700b3b35caddab2ab4a48a726a6cdbf78e0ce). 
OK i'm removing it for good.